### DISCUSSION TITLE
PCB Design Help

### DISCUSSION THREAD

**Discussion by everettmajor** [ekumanov/cybrid#6] [General]
Hi there,

I am building a similar project inspired by yours. I tried uploading the gerber files for fabrication for the Sensor PCB (I like your design!) but I got a warning message saying there were no drill holes. Did you run into any issues printing your PCB, or did they cut out your screw hole correctly?

Thanks!

**Comment by davidedelvento** [comment]
@everettmajor I saw you posted also on stem-piano but that post is now gone (perhaps you mistakenly posted to the wrong repo?)

Anyway, we recently placed the order as described in this other repo: https://github.com/stem-piano/orders/tree/main/PCBWay-nosensors-2024 and everything went fine -- we plan to place another (group) order soon. Do note that the project/design is DIFFERENT compared to Cybrid as described in various posts. You may be interested in looking at this comparison of various approaches: https://pianoclack.com/forum/d/745-summary-of-digitalhybrid-piano-diy-projects

**Reply by everettmajor** [reply]
I accidently commented on that post thinking I was on this one! If it worked for you, I feel comfortable placing that order. Thanks for those links, I'll take a look!
